packages/local_auth/android/src/main/java/io/flutter/plugins/localauth/AuthenticationHelper.java

```diff
@@ -92,8 +92,18 @@
     this.fingerprintManager = FingerprintManagerCompat.from(activity);
   }
 
+
+  boolean isFingerPrintAvailable(){
+    if(android.os.Build.VERSION.SDK_INT >= 23){
```

Comment:
This could be combined into a single return 
```
return ((android.os.Build.Version.SDK_INT >= 23) && 
        fingerprintManager.isHardwareDetected());
```